index.html

```diff
@@ -0,0 +1,141 @@
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+    rel="stylesheet">
+  <link rel="stylesheet" href="./styles/index.css">
+</head>
+
+<body>
+  <header class="header">
+    <div class="header__first-nav">
+      <h3 class="header__logo">Medical Core</h3>
+      <nav class="header__links-container">
+        <a href="#" class="header__link">Home</a>
+        <a href="#" class="header__link">Product</a>
+        <a href="#" class="header__link">Pricing</a>
+        <a href="#" class="header__link">Contact</a>
+      </nav>
+    </div>
+    <nav class="header__auth-links">
+      <a href="#" class="header__link header__link--green">Login</a>
+      <button class="button button--primary">JOIN US</button>
+    </nav>
+  </header>
+  <main>
+    <section class="hero">
+      <div class="section-text section-text--medium-gap">
+        <h4 class="section-text__title">Welcome</h4>
+        <h1 class="section-text__heading section-text__heading--big">A Great Place to Receive Care</h1>
+        <p class="section-text__description section-text__description--big">Get your best looking smile now!</p>
+      </div>
+      <div class="hero__buttons-container">
+        <button class="button button--primary button--wide">Join us</button>
+        <button class="button button--secondary button--wide">Learn more</button>
+      </div>
+    </section>
+
+    <section class="meet-section">
+      <div>
+        <img class="meet-section__image" src="./images/thumb-concept.png" alt="a female doctor holdin stethoscope">
+      </div>
+      <div>
+        <div class="meet-section__line"></div>
+        <div class="section-text section-text--long-gap">
+          <h2 class="section-text__heading">Meet our experts</h2>
+          <p class="section-text__description">Problems trying to resolve the conflict between
+            the two major realms of Classical physics:
+            Newtonian mechanics</p>
+          <h5 class="section-text__title">Learn more</h5>
+        </div>
+      </div>
+    </section>
+
+    <section class="team-section">
+      <div class="section-text">
+        <h4 class="section-text__title">Team</h4>
+        <h2 class="section-text__heading">Our team</h2>
+        <p class="section-text__description">Problems trying to resolve the
+          conflict between
+          the two major realms of Classical physics: Newtonian mechanics</p>
+      </div>
+      <div class="team-section__cards-container">
+        <div class="person-card">
+          <div><img class="person-card__image" src="./images/user-cover-1.png"
```

Comment:
You should give the div a class for stylistic purpose or get rid of the div since elements with no classes are not necessary.